webpack.config.js

```diff
@@ -2,6 +2,7 @@ const path = require('path');
 const TerserPlugin = require('terser-webpack-plugin');
 
 module.exports = {
+    mode: 'development',
```

Comment:
I had an PR locally to set it to `none` and forgot to open it. Please use `none` as we are forcing modes in script section in packages.json.